sfge/src/test/java/com/salesforce/graph/ops/expander/ApexPathExpanderTest.java

```diff
@@ -622,4 +624,67 @@ public void testInlineResolvableTwoLevelMethodCall() {
         MatcherAssert.assertThat(results, hasSize(equalTo(1)));
         MatcherAssert.assertThat(results, TestRunnerListMatcher.hasValuesAnyOrder("hello"));
     }
+
```

Comment:
I haven't had to add new tests since the existing ones covers ground for the changes I'm making. For the Registry framework portion, the usage of `PathExpansionRegistry` feels sufficient to me. Let me know if you can think of cases that don't exist.